---
 master.cf |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/master.cf.orig
+++ b/master.cf
@@ -9,7 +9,7 @@ Hunk #1, master.cf
 # service type  private unpriv  chroot  wakeup  maxproc command + args
 #               (yes)   (yes)   (no)    (never) (100)
 # ==========================================================================
-smtp      inet  n       -       n       -       -       smtpd
+#smtp      inet  n       -       n       -       -       smtpd
 #smtp      inet  n       -       n       -       1       postscreen
 #smtpd     pass  -       -       n       -       -       smtpd
 #dnsblog   unix  -       -       n       -       0       dnsblog
