---
 lvm.conf |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- lvm.conf.orig
+++ lvm.conf
@@ -298,7 +298,7 @@ Hunk #1, lvm.conf devices {
 	# benefit from discards, but SSDs and thinly provisioned LUNs
 	# generally do. If enabled, discards will only be issued if both the
 	# storage and kernel provide support.
-	issue_discards = 0
+	issue_discards = 1
 
 	# Configuration option devices/allow_changes_with_duplicate_pvs.
 	# Allow VG modification while a PV appears on multiple devices.
